activerecord/lib/active_record/test_databases.rb

```diff
@@ -20,13 +20,13 @@ def self.create_and_migrate(i, spec_name:)
       connection_spec["database"] += "-#{i}"
       ActiveRecord::Tasks::DatabaseTasks.create(connection_spec)
       ActiveRecord::Base.establish_connection(connection_spec)
-      ActiveRecord::Tasks::DatabaseTasks.migrate
+      ActiveRecord::Tasks::DatabaseTasks.load_schema(connection_spec)
     ensure
       ActiveRecord::Base.establish_connection(ActiveRecord::Base.configurations[Rails.env])
       ENV["VERBOSE"] = old
     end
 
-    def self.drop(i, spec_name:)
+    def self.drop(spec_name:)
```

Comment:
Should not the `i` be used here?

Reply:
It took me a couple of looks to see why this is okay: `create_and_migrate` modifies the original connection spec hash, so the rest of the (forked) app, including this method, sees that already-suffixed DB name.

Reply:
Yeah, that is confusing.